docs/en/quick-start.mdx

```diff
@@ -4,63 +4,93 @@ sidebar_label: Quick Start
 sidebar_position: 1
 keywords: [clickhouse, install, getting started, quick start]
 pagination_next: 'en/getting-started/index'
+title: "ClickHouse Quick Start"
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import CodeBlock from '@theme/CodeBlock';
 
-# ClickHouse Quick Start
+## Overview
 
-:::tip
-This page helps you set up open-source ClickHouse on your own machine. The fastest way to deploy ClickHouse and to get access to our exclusive SQL Console is to use ClickHouse Cloud.
+Get set up quickly with ClickHouse. Download an appropriate binary for your OS,
+learn to run ClickHouse server, and create a table, insert data into it, and
+query your table using ClickHouse client.
 
-New users get $300 in free trial credits. Click [here](https://clickhouse.cloud/signUp?loc=docs-quick-start) to sign up.
+### Prerequisites
+
+You'll need curl or another command-line HTTP client to fetch the ClickHouse
+binary.
+
+## Download the binary
+
+ClickHouse runs natively on Linux, FreeBSD and macOS, and runs on Windows via
+the [WSL](https://learn.microsoft.com/en-us/windows/wsl/about). The simplest way to download ClickHouse locally is to run the
+following `curl` command. It determines if your operating system is supported,
+then downloads an appropriate ClickHouse binary.
+
+:::note
+We recommend running the command below from a new and empty subdirectory as
+some configuration files will be created in the directory the binary is located
+in the first time ClickHouse server is run.
 :::
 
-##  1. Download the binary
+```bash
+curl https://clickhouse.com/ | sh
+```
+
+You should see:
 
-ClickHouse runs natively on Linux, FreeBSD and macOS, and runs on Windows via the [WSL](https://learn.microsoft.com/en-us/windows/wsl/about).
-The simplest way to download ClickHouse locally is to run the following `curl` command. It determines if your operating system is supported,
-then downloads an appropriate ClickHouse binary:
+```
+Successfully downloaded the ClickHouse binary, you can run it as:
+    ./clickhouse
```

Comment:
I'm not sure this makes sense in a "Quick Start" because this runs clickhouse-local, which will be confusing to someone new to ClickHouse.

Reply:
This is what gets output when you run the curl command so it's what users will see when they follow the quick-start. Agree that the prompt is confusing, but core team will need to change that if necessary.